src/features/Applet/Schedule/Calendar/Calendar.utils.ts

```diff
@@ -5,3 +5,6 @@ export const getEventsWithOffRange = (events: CalendarEvent[], date: Date) =>
     ...event,
     isOffRange: event.start.getMonth() !== date.getMonth(),
   }));
+
+export const formatToYearMonthDate = (date?: Date) =>
+  date && [date.getFullYear(), date.getMonth(), date.getDate()].join('-');
```

Comment:
can we use [date-fns format](https://date-fns.org/v1.29.0/docs/format) here?

Reply:
done